habitat-lab/habitat/tasks/rearrange/utils.py

```diff
@@ -388,6 +388,54 @@ def write_gfx_replay(gfx_keyframe_str, task_config, ep_id):
         text_file.write(gfx_keyframe_str)
 
 
+def place_agent_at_dist(
```

Comment:
why is this `place_agent_at_dist` and not `place_agenta_at_pos`?

Reply:
Sure, I will rename to `place_agent_at_dist_from_pos`.